src/autopas/LogicHandler.h

```diff
@@ -202,7 +202,7 @@ class LogicHandler {
     for (size_t i = 0; i < newLength.size(); ++i) {
       // warning threshold is set arbitrary and up for change if needed
       if (relDiffLength[i] > 1.3 or relDiffLength[i] < 0.7) {
-        AutoPasLog(WARN,
+        AutoPasLog(DEBUG,
```

Comment:
Again, never actually submitted my reasoing!

> In my experience, this gets quite spammy. This message is only interesting in the case that a tuning strategy which is based purely on past iterations (e.g. predictive tuning) is used. In such a case, the only consequence is a potentially suboptimal configuration.

Reply:
Ok, if you think this is a solved problem, it doesn't need to be a warning.

When you write your next tuning paper be sure to include a chapter that shows how different tuning strategies handle such rapidly changing (sub)domains.

Reply:
No I never said this was a solved problem. But it is a rather spammy message that isn't useful for strategies that aren't based on past information (e.g. rules, fuzzy, decision trees) and for those which are (e.g. predictive)) it serves less as a warning (e.g. this is undefined behaviour and potentially wrong like fast particles) and more as an explanation for potential poor performance. Hence why I felt that `DEBUG` was more appropriate.

Ultimately, even though I don't think this should be an AutoPas warning, I don't really care about keeping this as a warning because if this warning is going off, it usually means I've picked bad load-balancing options and that's good to know.

Reply:
> an explanation for potential poor performance

Hence why I would keep it as `WARN`. I fear I'd just not check every scenario with `DEBUG` just to make sure this doesn't happen unnoticed.

>  if this warning is going off, it usually means I've picked bad load-balancing options and that's good to know.

Literally my point why I think it is important to keep visible :)

> Ultimately, even though I don't think this should be an AutoPas warning

I see your point that it is not a warning that something IN AutoPas goes wrong but IMHO it is a warning that AutoPas is used in a potentially unsafe way.

> isn't useful for strategies that aren't based on past information

Fully agree here. The question is just do we want to introduce a mechanism that only fires this message if such a strategy is active (no a hardcoded if is not a futureproof option) or do we always fire. We can potentially amend the message that this only affects strategies that rely on past iterations.